lib/config/index.js

```diff
@@ -96,11 +97,21 @@ config.isGitLabEnable = config.gitlab.clientID && config.gitlab.clientSecret
 config.isMattermostEnable = config.mattermost.clientID && config.mattermost.clientSecret
 config.isLDAPEnable = config.ldap.url
 config.isSAMLEnable = config.saml.idpSsoUrl
-config.isPDFExportEnable = config.allowpdfexport
+config.isPDFExportEnable = config.allowPDFExport
 
 // merge legacy values
-if (config.imageUploadType && !config.imageuploadtype) {
-  config.imageuploadtype = config.imageUploadType
+let keys = Object.keys(config)
+for (let i = keys.length; i--;) {
+  const uppercase = /[A-Z]/
```

Comment:
`const uppercase....` can be taken out of the loop.

Reply:
:+1: